packages/trader/src/App/__tests__/init-store.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { setWebsocket } from '@deriv/shared';
+import { mockStore } from '@deriv/stores';
+import initStore from 'App/init-store';
+
+const mock_ws = {
+    activeSymbols: jest.fn(),
+    authorized: {
+        activeSymbols: jest.fn(),
+        subscribeProposalOpenContract: jest.fn(),
+        send: jest.fn(),
+    },
+    buy: jest.fn(),
+    storage: {
+        contractsFor: jest.fn(),
+        send: jest.fn(),
+    },
+    contractUpdate: jest.fn(),
+    contractUpdateHistory: jest.fn(),
+    subscribeTicksHistory: jest.fn(),
+    forgetStream: jest.fn(),
+    forget: jest.fn(),
+    forgetAll: jest.fn(),
+    send: jest.fn(),
+    subscribeProposal: jest.fn(),
+    subscribeTicks: jest.fn(),
+    time: jest.fn(),
+    tradingTimes: jest.fn(),
+    wait: jest.fn(),
+};
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    setWebsocket: jest.fn(),
+}));
+
+describe('initStore', () => {
+    it('should return the root store', () => {
+        const root_store = mockStore({});
```

Comment:
we can declare it outside (after `describe('initStore', () => {` ) since it's used in every test case and not mutated. :) please use camelCase for new variables 🙏🏻
```
        const rootStore = mockStore({});
```